### PR TITLE
OHRM5X-1141 Test Automation: Admin - Employment Status - API

### PR DESCRIPTION
**Pull request by ShenaliFernando**

### CODE REVIEW

File: symfony/test/functional/cypress/integration/api/admin/empStatus_API.spec.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+      cy.apiLogin(admin);
+    });
+  });
+
+  describe('POST /emp-Status', function () {
+    it('create a new employment status', function () {
+      cy.request('POST', empStatusAPI, {
+        name: this.strings.chars50.text,
+      }).then((response) => {
+        console.log(response);
```
Comment (Super-Chama) [inline symfony/test/functional/cypress/integration/api/admin/empStatus_API.spec.js:17]: remove `console.log()`. run `yarn lint` for clean up.

File: symfony/test/functional/cypress/integration/api/admin/empStatus_API.spec.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+  describe('PUT /emp-Status', function () {
+    it('update an employment status', function () {
+      cy.task('db:restore', {name: 'emp-Status-POST1'});
+      cy.request('GET', empStatusAPI).then((response) => {
+        const statuslist = response.body.data.map((item) => item.id);
+        console.log(response);
+        expect(response.status).to.eq(200);
+        return cy
+          .request('PUT', empStatusAPI + '/' + statuslist[0], {
```
Comment (Super-Chama) [inline symfony/test/functional/cypress/integration/api/admin/empStatus_API.spec.js:84]: `statuslist[0]` is nicer than hard coding. :+1: 
can you update my [job title spec ](https://github.com/orangehrm/orangehrm/blob/5.0/symfony/test/functional/cypress/integration/api/admin/jobTitles_API.spec.js)also?

File: symfony/test/functional/cypress/integration/api/admin/empStatus_API.spec.js (hunk 1 above)
Comment (ShenaliFernando) [reply symfony/test/functional/cypress/integration/api/admin/empStatus_API.spec.js]: Done.

File: symfony/test/functional/cypress/integration/api/admin/empStatus_API.spec.js (hunk 2 above)
Comment (ShenaliFernando) [reply symfony/test/functional/cypress/integration/api/admin/empStatus_API.spec.js:82]: Sure.

File: symfony/test/functional/cypress/integration/api/admin/empStatus_API.spec.js [hunk 3]
```diff
@@ -0,0 +1,92 @@
+const empStatusAPI = '/api/v2/admin/employment-statuses';
```
Comment (RajithaKumara) [inline symfony/test/functional/cypress/integration/api/admin/empStatus_API.spec.js:1]: batter to add copyright header.

File: symfony/test/functional/cypress/integration/api/admin/empStatus_API.spec.js [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+
+  describe('GET /emp-Status ', function () {
+    it('get all employment status', function () {
+      cy.request('GET', empStatusAPI).then((response) => {
+        expect(response.status).to.eq(200);
+        expect(response.body.data.length).to.eq(0);
+      });
+    });
+    it('get one employment status', function () {
+      cy.task('db:restore', {name: 'emp-Status-POST1'});
```
Comment (RajithaKumara) [inline symfony/test/functional/cypress/integration/api/admin/empStatus_API.spec.js:32]: I think this is again rely on POST test cases

**Review by RajithaKumara — CHANGES_REQUESTED** [review]

**Comment by RajithaKumara** [comment]
Closing since moved to #1159

**Pull request closed without merging**